terraform/aws/analytical-platform-data-production/airflow/eks.tf

```diff
@@ -1,4 +1,11 @@
 resource "aws_eks_cluster" "airflow_dev_eks_cluster" {
+  encryption_config {
+    resources = ["secrets"]
+    provider {
+      key_arn = aws_kms_key.eks.arn
+    }
+  }
+
```

Comment:
In this case, I agree that encryption is missing and should be added if feasible. Would like to test in sandpit ideally first? I know that's not in terraform though, so would be somewhat manual testing.

Reply:
yes lets test in sandpit